src/main/java/uk/gov/hmcts/probate/transformer/CaveatCallbackResponseTransformer.java

```diff
@@ -153,7 +155,8 @@ public CaveatCallbackResponse addMatches(CaveatCallbackRequest request, List<Cas
     public CaveatCallbackResponse transformResponseWithExtendedExpiry(CaveatCallbackRequest caveatCallbackRequest) {
```

Comment:
The method transformResponseWithExtendedExpiry needs a unit test.

Reply:
added